sensor_msgs/Image img_processed
bbox[] batchBbox